src/backend/distributed/shared_library_init.c

```diff
@@ -876,6 +876,16 @@ RegisterCitusConfigVariables(void)
 		GUC_SUPERUSER_ONLY | GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_mx_by_default",
```

Comment:
Let's maybe:
- call it as more generic, something like sync_metadata_to_nodes or such?
- Make it superuser only GUC and Hidden GUC